Tests/CoreTests/ReflectableTests.swift

```diff
@@ -52,6 +52,31 @@ class ReflectableTests: XCTestCase {
         try XCTAssert(Foo.reflectProperty(forKey: \.odir)?.type is Direction?.Type)
     }
 
+    func testCaseIterableExtension() throws {
+        #if swift(>=4.2)
+        enum Pet: String, CaseIterable, ReflectionDecodable, Decodable {
+            case cat
```

Comment:
How is this test working if there is only one case?

Reply:
Oops, deleted to to check for fail but forgot to add it back 😨

Reply:
You might want to have 2 enums, you to test for failure.

Reply:
👍 Yeah, added